style.css

```diff
@@ -75,3 +76,10 @@ canvas {
   align-self: center;
   margin: 0 10px;
 }
+
+@media only screen and (max-width: 540px) {
```

Comment:
Nice! Not necessarily something that needs to be addressed this PR but we might want to look at adding a media queries to address tablet devices. Maybe a few if there are different layouts we prefer for device screen sizes 🤷🏽‍♂️. An enhancement like this could be a good card to add to the projects `To do` column. What do you think?

Reply:
Yea definitely, I'd like to add that to be able to use it in different screen sizes. 👍